src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -62,9 +76,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             array('Environment', $kernel->getEnvironment()),
             array('Debug', $kernel->isDebug() ? 'true' : 'false'),
             array('Charset', $kernel->getCharset()),
-            array('Root directory', self::formatPath($kernel->getRootDir(), $kernel->getProjectDir())),
-            array('Cache directory', self::formatPath($kernel->getCacheDir(), $kernel->getProjectDir()).' (<comment>'.self::formatFileSize($kernel->getCacheDir()).'</>)'),
-            array('Log directory', self::formatPath($kernel->getLogDir(), $kernel->getProjectDir()).' (<comment>'.self::formatFileSize($kernel->getLogDir()).'</>)'),
+            array('Root directory', self::formatPath($container->getParameter('kernel.root_dir'), $projectDir)),
+            array('Cache directory', self::formatPath($cacheDir = $container->getParameter('kernel.cache_dir'), $projectDir).' (<comment>'.self::formatFileSize($cacheDir).'</>)'),
+            array('Log directory', self::formatPath($logDir = $container->getParameter('kernel.logs_dir'), $projectDir).' (<comment>'.self::formatFileSize($logDir).'</>)'),
```

Comment:
Why is it better to use parameters?

Reply:
nothing really, it's just that `getProjectDir` is not part of `KernelInterface`, and `$kernel->getContainer()->getParameter('kernel.some_path')` works for all paths.

See also https://github.com/symfony/symfony/pull/23624#issuecomment-318813073 where we favored `getParam` over injection.

Reply:
I don't like it. I don't buy the "it is not part of KernelInterface". Using a string sounds much worse than using a method.

Reply:
Reverted.